app/models/miq_worker/systemd_common.rb

```diff
@@ -202,5 +130,9 @@ def unit_environment_variables
         "HOME=/root"
       ]
     end
+
+    def systemd_unit_dir
+      Pathname.new("/usr/lib/systemd/system")
```

Comment:
Why is this dir different than https://github.com/ManageIQ/manageiq/pull/20983/files#diff-67e01e3b1ea2575267fa04629e3e39801735cd8bac18b3608c1108e5fe3c602bR42 ?

Reply:
:+1: should be `/lib/systemd/system`